src/ctia/auth/capabilities.clj

```diff
@@ -1,17 +1,52 @@
 (ns ctia.auth.capabilities
-  (:require
-   [ctia.entity.entities :refer [entities]]))
+  (:require [clojure.set :as set]))
+
+(def entities
+  #{:actor
+    :attack-pattern
+    :campaign
+    :casebook
```

Comment:
casebook was not present on purpose in the entities list in the jwt ns.
So with this code this will make people with private-intel scope be able to access the casebook route which is incorrect.